omod/src/main/webapp/portlets/manageReportQueue.jsp

```diff
@@ -17,17 +17,9 @@
 		});
 	});
 	
-	function cancelReportRequest(id){
-		if(id && confirm('<spring:message code="reporting.reportRequest.cancel.confirm" />')){
-			DWRReportingService.purgeReportRequest(id, function(success){
-				if(success == true){
-					pos = reportRequestsTable.fnGetPosition(document.getElementById('reportRequest_'+id));
-					if(pos != undefined)
-						reportRequestsTable.fnDeleteRow(pos);
-				}else{
-					alert('<spring:message code="reporting.reportRequest.error.failedToCancel" />');
-				}
-			});
+	function cancelReportRequest(uuid){
+		if(uuid && confirm('<spring:message code="reporting.reportRequest.cancel.confirm" />')){
+			document.location.href = '${pageContext.request.contextPath}/module/reporting/reports/deleteReportRequest.form?uuid=' + uuid + '&returnUrl=${returnUrl}';
```

Comment:
That's what I was talking about in OpenMRS talk but I went with this way because it's the common practice. I didn't wanna introduce using AJAX for this just at this single point.